src/os/bluestore/BlueFS.cc

```diff
@@ -2292,3 +2292,10 @@ int BlueFS::unlink(const string& dirname, const string& filename)
   _drop_link(file);
   return 0;
 }
+
+bool BlueFS::wal_is_rotational()
+{
+  if (!bdev[BDEV_WAL] || bdev[BDEV_WAL]->is_rotational())
```

Comment:
@jdurgin This check is only for the journal. We will be checking both `is_rotational` and `is_journal_rotational` in the osd, before deciding which value of recovery sleep to use. I will be adding that logic in my next commit. Does that sound reasonable?